src/Application.cc

```diff
@@ -570,7 +570,8 @@ std::vector<std::pair<std::string, std::vector<std::string>>>
   std::string home;
   common::env(IGN_HOMEDIR, home);
   paths.push_back(home + "/.ignition/gui/plugins");
-
+  paths.push_back(common::cwd() + "/" + std::string(IGN_GUI_PLUGIN_PATH));
```

Comment:
I'm not sure what the intention is here, but the plugins installed with `ign-gui` are meant to be covered by the case below. Isn't Bazel installing them to `IGN_GUI_PLUGIN_INSTALL_DIR`?

Another thing to consider when adding plugin paths is that Ignition GUI will display all shared libraries it finds on those paths under the top-right menu. So we don't want to be adding paths where non-GUI plugins can also be found.